### ISSUE TITLE
Adds support for default attributes

### ISSUE THREAD

**Issue by jcchavezs** [hypertrace/agent-config#21]
A common use case is that users can pass default attributes to instrumentation at different levels, e.g.:

- https://github.com/openzipkin/brave/blob/f89723df407e2b85dd7e6700ab574b4fa5bdae94/brave/src/test/java/brave/features/handler/DefaultTagsTest.java#L32
- https://github.com/openzipkin/zipkin-go/blob/master/middleware/http/server.go#L42
- https://github.com/openzipkin/zipkin-js/issues/502
- https://github.com/openzipkin/zipkin-js/pull/302

We can allow static tags being passed directly from code and also over env vars:

```proto
message AgentConfig {
    ...

    // defaultTags accept a map of tags that will be added on every span
    map<string, string> defaultTags = 4;
}

```

**Comment by pavolloffay** [comment]
LGTM, for the name I would consider using attributes, but the HT uses tags in the search bar so I am not sure.

**Comment by davexroth** [comment]
agreed, we should standardize on attributes.  And I thought we had gone through and updated HT to use attributes, but maybe we missed some places.  We should fix that.

**Comment by jcchavezs** [comment]
Yeah, makes sense to use attributes.
